SAMtags.tex

```diff
@@ -250,29 +250,23 @@ \subsection{Barcodes}
 The {\tt BC} tag should match the {\tt QT} tag in length. 
 In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and places a hyphen (`{\tt -}') between the barcodes from the same template. 
 
-\item[BZ:Z:\tagvalue{qualities+}] 
-Phred quality of the (uncorrected) unique molecular identifier sequence in the {\tt OX} tag.
+\item[QT:Z:\tagvalue{qualities}] 
+Phred quality of the sample-barcode sequence in the {\tt BC} (or {\tt RT}) tag. 
 Same encoding as {\sf QUAL}, i.e., Phred score + 33.
-The {\tt OX} tags should match the {\tt BZ} tag in length. 
-In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the quality strings with a space (`{\tt \textvisiblespace}') between the different strings.
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the quality strings with spaces (`{\tt \textvisiblespace}') between the different strings from the same template. 
 
 \item[CB:Z:\tagvalue{str}]
-  Cell identifier based on the (possibly corrected) cellular-barcode
-  sequence ({\tt CR}). The sequence may optionally be followed by a
-  dash ('-') and one or more alphanumeric characters to form an
-  identifier. The respective quality scores are (optionally) stored in
-  the {\tt CY} tag. All reads with the same {\tt CB} tag putatively
-  derive from the same cell.
+Cell identifier based on the cellular-barcode sequence ({\tt CR}), with a correction optionally applied. The sequence may optionally be followed by a dash (`{\tt -}') and one or more alphanumeric characters to form an identifier. All reads with the same {\tt CB} tag putatively derive from the same cell.
+In the case of the cell identifier being based on multiple cellular-barcode sequences the recommended implementation concatenates all the barcodes with a hyphen (`{\tt -}') between the different barcodes.
```

Comment:
shouldn't this sentence (about how to concatenate) be in the "Cellular Barcode" part, rather that the elular identifier part?

Reply:
Good point, makes more sense in the CR and CY. Changed the text accordingly.